mypy/checkmember.py

```diff
@@ -40,8 +40,10 @@ def analyze_member_access(name: str,
       1. lvalue or non-lvalue access (i.e. setter or getter access)
       2. supertype access (when using super(); is_super == True and
          override_info should refer to the supertype)
+
+    actual_self is the type of E in the expression E.foo
```

Comment:
I tried elaborating (below), but I seem to have problems phrasing it.

I don't know if there's a case where the types differ initially. It might have been the case for intersection :P